ui/src/app/pages/workspace/workspace-share.tsx

```diff
@@ -143,6 +150,72 @@ export const UserRoleOptions = [
   { value: WorkspaceAccessLevel.OWNER, label: 'Owner' },
 ];
 
+interface ConflictProps {
+  reloadAction: () => void;
+  cancelAction: () => void;
+}
+const ConflictModal = (props: ConflictProps) => (
```

Comment:
not strictly related, but it was a helpful pattern to follow after extracting this from the main component